### PR TITLE
Completed React Assessment

### PR DESCRIPTION
**Pull request by kimberlyleo**

### CODE REVIEW

File: react-commenting.js [hunk 1]
```diff
@@ -60,9 +61,10 @@ class Board extends Component{
       [2, 4, 6],
     ]
     winningConditions.map(value => {
+      // I don't inderstand the code below.. maybe we could go over this in class if we have time
       const [a, b, c] = value
```
Comment (sjproctor) [inline react-commenting.js:65]: We are destructuring the array and creating variables to see if they all hold the same value in the spot on the gameboard.

File: react.js [hunk 2]
```diff
@@ -17,7 +30,9 @@ for(let i=0; i<names.length; i++){
   console.log(`${names[i]} is ${names[i].length} characters long.`)
 }
 
-
+names.map((v,i) => {
+  console.log(`${names[i]} is ${names[i].length} characters long.`)
+})
```
Comment (sjproctor) [inline react.js:36]: ${names[i]} can be written as ${v}

**Review by sjproctor — APPROVED** [review]
Yay looks great!

File: react.js (hunk 2 above)
Comment (kimberlyleo) [reply react.js:36]: great. thanks!!

**Pull request merged**